src/README.md

```diff
@@ -1,31 +1,65 @@
+  - [J-Link GDB Server](#jlink-gdb-server)
+  - [J-Link Commander](#jlink-commander)
+- [Run the example with EFR32 boards](#example)
+- [Additional features](#additional-features)
+- [Verification](#verification)
+
+---
 
 This directory contains example platform drivers for the [Silicon Labs EFR32MG12][efr32mg] based on [EFR32™ Mighty Gecko Wireless Starter Kit][slwstk6000b] or [Thunderboard™ Sense 2 Sensor-to-Cloud Advanced IoT Development Kit][sltb004a].
 
-[efr32mg]: http://www.silabs.com/products/wireless/mesh-networking/efr32mg-mighty-gecko-zigbee-thread-soc
+[efr32mg]: https://www.silabs.com/support/getting-started/mesh-networking/thread/mighty-gecko
 [slwstk6000b]: http://www.silabs.com/products/development-tools/wireless/mesh-networking/mighty-gecko-starter-kit
 [sltb004a]: https://www.silabs.com/products/development-tools/thunderboard/thunderboard-sense-two-kit
 
-The example platform drivers are intended to present the minimal code necessary to support OpenThread. [EFR32MG12P SoC][efr32mg12p] has rich memory and peripheral resources which can support all OpenThread capabilities. See the "Run the example with EFR32MG12 boards" section below for an example using basic OpenThread capabilities.
+The example platform drivers are intended to present the minimal code necessary to support OpenThread. [EFR32MG][efr32mg] has rich memory and peripheral resources which can support all OpenThread capabilities.
 
-See [sleepy-demo/README.md](sleepy-demo/README.md) for instructions for an example that uses the low-energy modes of the EFR32MG12 when running as a Sleepy End Device.
+See [EFR32 Sleepy Demo Example](./sleepy-demo/README.md) for instructions for an example that uses the low-energy modes of the EFR32MG12 when running as a Sleepy End Device.
 
-[efr32mg12p]: http://www.silabs.com/products/wireless/mesh-networking/efr32mg-mighty-gecko-zigbee-thread-soc/device.EFR32MG12P432F1024GL125
+<a name="prerequisites"/>
 
-## Toolchain
+## Prerequisites
+
+<a name="toolchain"/>
+
+### Toolchain
 
 Download and install the [GNU toolchain for ARM Cortex-M][gnu-toolchain].
 
 [gnu-toolchain]: https://developer.arm.com/open-source/gnu-toolchain/gnu-rm
 
-In a Bash terminal, follow these instructions to install the GNU toolchain and other dependencies.
+In a `bash` terminal, follow these instructions to install the GNU toolchain and other dependencies.
 
 ```bash
 $ cd <path-to-ot-efr32>
 $ ./script/bootstrap
 ```
 
-## Build Examples
+<a name="flash-debug"/>
+
+## Flashing and debugging tools
+
+Install [Simplicity Studio][simplicity-studio] to flash, debug, and make use of logging features with SEGGER J-Link.
+
+[simplicity-studio]: https://www.silabs.com/developers/simplicity-studio
+
+Alternatively, the [J-Link][jlink-software-pack] software pack can be used to flash and debug.
+
+[jlink-software-pack]: https://www.segger.com/downloads/jlink/#J-LinkSoftwareAndDocumentationPack
+
+<a name="build"/>
+
+## Building examples
```

Comment:
I ran the "./script/build efr32mg12 -DBOARD=brd4161a" has defined in this README after running ./script/bootstrap,.  It appeared to build eveerything but failed at the end with the following and no .s37 files were created:

> EXECUTABLE='-perm +111'
> find /Users/romacdon/projects/silabs-ot-efr32/build/efr32mg12 -type f '-perm +111' -not -name '*.*' -exec arm-none-eabi-objcopy -O srec '{}' '{}.s37' ';' -exec ls -alh '{}' '{}.s37' ';'
> find: -perm +111: unknown primary or operator

Reply:
Fixed with a7528510c8b725dbd94b598d0f6fe2350ab7ca34